grpc-ingest/README.md

```diff
@@ -1,33 +1,71 @@
-## Dev process
+## Dev setup
 
-### geyser gRPC source
+> **Note:** Run these commands from the root of the project
 
-Use [Triton One](https://triton.one/) provided endpoint or run own node with geyser plugin: https://github.com/rpcpool/yellowstone-grpc
-
-### Redis server
+### Run redis, postgres and prometheus docker containers
 
+```bash
+docker compose up db redis prometheus
 ```
-redis-server
+
+### Seed the database and run migrations
+
+```bash
+INIT_FILE_PATH=./init.sql sea migrate up --database-url=postgres://solana:solana@localhost:5432/solana
 ```
 
-### PostgreSQL server
+### Config for grpc2redis [./config-grpc2redis.yml](./config-grpc2redis.yml)
 
-Run:
+change endpoint and x-token in the `geyser` section
 
-```
-docker run -it --rm -e POSTGRES_PASSWORD=solana -e POSTGRES_USER=solana -e POSTGRES_DB=solana -p 5432:5432 postgres
+multiple account and transaction subscriptions can be added in the `subscriptions` section
+filter shoud be compatible with the geyser gRPC filters depending on account or transaction
+
+> **Note:** Only Bubblegum transaction instructions are handled by the ingester for now
+
+```yaml
+subscriptions:
+  token-metadata:
+    stream:
+      name: ACCOUNTS
+      max_len: 100_000_000
+      max_concurrency: 2
+    filter:
+      accounts:
+        owner:
+          - metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s
+  bubblegum:
+    stream:
+      name: TRANSACTIONS
+      max_len: 100_000_000
+      max_concurrency: 2
+    filter:
+      transactions:
+        account_include:
+          - BGUMAp9Gq7iTEuizy4pqaxsTyUCBK68MDfK752saRPUY
```

Comment:
Can you please add a section about prometheus? The fact that each command starts a prometheus server and you can access the metrics at `http://0.0.0.0:{port}/metrics`.